s4e-web/cypress/plugins/index.js

```diff
@@ -35,3 +35,17 @@ module.exports = (on, config) => {
     }
   });
 };
+
+const path = require('path')
+
+module.exports = (on) => {
```

Comment:
If you're using some walk around's write why you're using it. The shorter and simpler description, the better.